test/e2e/cli_with_server_test.go

```diff
@@ -196,6 +196,42 @@ func (s *CLIWithServerSuite) TestTemplateLevelSemaphore() {
 		DeleteConfigMap()
 }
 
+func (s *CLIWithServerSuite) TestSynchronizationWfLevelMutex() {
+	s.testNeedsOffloading()
```

Comment:
why does this need offloading? surely it will work without?

Reply:
remove it